bigtable/admin.go

```diff
@@ -671,6 +671,18 @@ func storageTypeFromProto(st btapb.StorageType) StorageType {
 	return SSD
 }
 
+// InstanceState is the state of the instance. This is output-only.
+type InstanceState int32
+
+const (
+	// NOT_KNOWN is state of the instance could not be determined
+	NOT_KNOWN InstanceState = InstanceState(btapb.Instance_STATE_NOT_KNOWN)
```

Comment:
MixedCaps: https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps

```suggestion
	NotKnown InstanceState = InstanceState(btapb.Instance_STATE_NOT_KNOWN)
```